components/admin_console/permission_schemes_settings/permissions_tree/permissions_tree.jsx

```diff
@@ -79,6 +79,7 @@ export default class PermissionsTree extends React.PureComponent {
                         ],
                     },
                     Permissions.DELETE_PUBLIC_CHANNEL,
+                    Permissions.CONVERT_PUBLIC_CHANNEL,
```

Comment:
CONVERT_PUBLIC_CHANNEL for me sounds not very explicit. I would prefer something like `CONVERT_PUBLIC_CHANNEL_TO_PRIVATE` or something like that.